package.json

```diff
@@ -16,7 +16,8 @@
         "test": "vitest",
         "test:coverage": "vitest --watch=false --coverage",
         "lint": "eslint . --ext .ts",
-        "fmt": "prettier . --write"
+        "fmt": "prettier . --write",
```

Comment:
❓ should we add `docs` to the `.prettierignore` file?

Reply:
good call :+1:

Reply:
https://github.com/fp-toolkit/fp-toolkit/pull/33/commits/bb3e5d2bce5baf74a293fc83a2c069d53b9decb3